roles/ciao-controller/defaults/main.yml

```diff
@@ -56,6 +56,9 @@ ciao_openstack_services:
   - service: cinderv2
     type: volumev2
     description: CIAO Storage Service v2
+  - service: glance
```

Comment:
cinder apis where created in the controller role because is the controller binary who exposes the storage api.

I think these services and endpoints should be created in the ciao-image role